packages/platform-ios/src/commands/logIOS/index.ts

```diff
@@ -38,7 +38,7 @@ async function logIOS() {
   };
 
   const device = findAvailableDevice(devices);
-  if (device === null) {
+  if (!device) {
```

Comment:
This can be either `null` or an object (see line 18 where you assert on `device` being able to access keys like `availability`). Can we please leave the check as it was?